docs/development/local-plugin-creation.md

```diff
@@ -0,0 +1,292 @@
+# Local plugin development guide
+
+So you wanna build a BitOps plugin, eh?  Follow along for how to do it locally!
+
+## 1. Create a plugin repo
+```
+cd /path/to/bitops-plugins
+mkdir my-plugin
+cd my-plugin
+```
+More information on what goes in a plugin [here](../plugins.md).
+
+> Note: If the plugin needs to install tools, you will need to build and run a local version of BitOps.  The first portion of this guide assumes the install.sh script **DOES NOT** needs to be run.  For more information about how to set up BitOps for local development with a plugin that requires an installation, skip to step 5 below.
+
+
+For this example, we'll keep to a really simple plugin:
```

Comment:
I'd replace this with a simple;
To start with, create a file called `bitops.schema.yaml` and add the following content.

The intention being that the documentation should more or less read the same across the pages.